templates/core/core-dpl.yaml

```diff
@@ -38,6 +38,45 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+        - name: "install-certificates"
+          securityContext:
+            runAsUser: 0
+          image: {{ .Values.core.image.repository }}:{{ .Values.core.image.tag }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: ["/bin/sh"]
+          args: [
+             '-c',
+             '/usr/bin/harbor-install-cert.sh &&
+             cp -ar /var/lib/ca-certificates/* /tmp/ca-certificates' ]
+          volumeMounts:
+          - name: ca-certificates
+            mountPath: /tmp/ca-certificates
+            readOnly: false
+          - name: secret-key
+            mountPath: /etc/core/key
+            subPath: key
+          - name: token-service-private-key
```

Comment:
I don't think this is needed either. The certificate installation script only uses `/etc/harbor/ssl` (internal certificates) and `/harbor_cust_cert` (set up by the `caBundleVolumeMount` template) as input paths.